addons/web_editor/static/src/xml/editor.xml

```diff
@@ -258,6 +258,19 @@
                         </select>
                     </div>
                 </div>
+                <div class="form-group">
+                    <label class="control-label col-sm-3">Style</label>
+                    <div class="col-sm-9">
+                        <select name="link_style_monkey" class="form-control col-sm-3 link-style">
+                            <option value="3d">3D</option>
+                            <option value="flat">Flat</option>
+                            <option value="" selected="selected">Default</option>
+                            <option value="outline">Outline</option>
+                            <option value="rounded">Rounded</option>
+                            <option value="rounded_outline">Outline &amp; Rounded</option>
```

Comment:
Definitely something we need to do after going to bootstrap 4, as it is standard in there